gocourse02/zoo/tiger.go

```diff
@@ -0,0 +1,28 @@
+package zoo
+
+import (
+	"fmt"
+	"reflect"
+)
+
+type Tiger struct {
+	*Animal
+	Age int
+}
+
+func NewTiger(animal *Animal, age int) *Tiger {
+	return &Tiger{
+		Animal: animal,
+		Age:    age,
+	}
+}
+
+func (t *Tiger) Reproduce(otherTiger *Tiger) *Tiger {
```

Comment:
Вище вже писав для `Snake` схожий комент:
```suggestion
func (t *Tiger) Reproduce(tiger *Tiger) *Tiger {
```